bosh-director/spec/unit/instance_deleter_spec.rb

```diff
@@ -273,6 +273,25 @@ module Bosh::Director
             end
           end
         end
+
+        context 'when virtual_delete_vm option is passed in' do
+          let(:options) { {virtual_delete_vm: true} }
+
+          it 'deletes snapshots, persistent disk, releases old reservations, vm should not delete from cloud' do
```

Comment:
I'm able to completely remove the implementation that this test corresponds with and this test still passes. Are you sure it's testing what you want it to test?